internal/sqlparser/testdata/valid-up/test06/03.golden.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO article (id, content) VALUES ('id_0002',  E'# My second 
```

Comment:
We partially fixed this issue, but there was an outstanding problem with the comments that users may want to preserve, see https://github.com/pressly/goose/pull/372#issuecomment-1182643810